docs/installation/basic_installation.md

```diff
@@ -58,7 +59,37 @@ If you would prefer less automation, or you  already have existing services on y
 
     <p>Once the preparations are done, the installer will prompt you to run <code>seat install:production</code>. </p>
 
-    <p>To configure ESI for SSO and API pulls, please refer to the <a href="/configuration/esi_configuration/">Configuring ESI</a> documentation page</p>
+</div>
+</section>
+
+!!! success "Installation complete"
+
+    You now have SeAT installed.
+    
+## Next steps
+
+1. [Configure ESI](/configuration/esi_configuration/) for SSO and API pulls
+2. Create a SeAT account with a character of yours
+3. Log off or use new browser
+4. Create admin link
+
+<section class="mdc-tabs">
```

Comment:
Please can we convert to linkable sections instead of tabs here. Going to get rid of tabs sometime in the future. Thanks!